allure-cucumber4-jvm/src/main/java/io/qameta/allure/cucumber4jvm/AllureCucumber4Jvm.java

```diff
@@ -40,12 +40,7 @@
 
 import java.io.ByteArrayInputStream;
 import java.nio.charset.Charset;
-import java.util.Deque;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
revert this change

Reply:
done + fixed a few pmd and checkstyle warnings